rest-servlet/src/main/java/org/hawkular/inventory/rest/RestTenants.java

```diff
@@ -100,4 +134,20 @@ public Response delete() {
         return Response.noContent().build();
     }
 
+    @DELETE
+    @Path("/")
+    @ApiOperation("Deletes the tenant and all its data. Be careful!")
+    @ApiResponses({
+                          @ApiResponse(code = 204, message = "OK"),
+                          @ApiResponse(code = 401, message = "Unauthorized access"),
+                          @ApiResponse(code = 404, message = "Tenant doesn't exist", response = ApiError.class),
+                          @ApiResponse(code = 500, message = "Server error", response = ApiError.class)
+                  })
+    public Response delete(@PathParam("tenantId") String tenantId) {
```

Comment:
Good. We could use the serialized canonical path form from my upcoming changes then..

Both for stable ids for authz and for json-ld stuff.

I think having a real URL in json-ld is still somewhat desired so we may introduce a new endpoint `/element?id=...` where the `...` would be the serialized form of the canonical path.

The serialized form right now looks like:
`t|tenantId/e|envId/f|feedId/r|parentResource/r|childResource`. We could change that so that it is more easily passed as a query parameter (not sure if the slashes would cause problem in a query param value or not).